lightstep/resource_metric_condition_test.go

```diff
@@ -363,8 +364,8 @@ func testAccCheckMetricConditionExists(resourceName string, condition *client.Me
 			return fmt.Errorf("ID is not set")
 		}
 
-		client := testAccProvider.Meta().(*client.Client)
-		cond, err := client.GetMetricCondition(context.Background(), test_project, tfCondition.Primary.ID)
+		providerClient := testAccProvider.Meta().(*client.Client)
+		cond, err := providerClient.GetMetricCondition(context.Background(), test_project, tfCondition.Primary.ID)
```

Comment:
This is to make linter happy. `client` was colliding with an import. Renaming makes linter happy.